src/cljs/main/broadfcui/page/groups/groups_management.cljs

```diff
@@ -0,0 +1,132 @@
+    [broadfcui.common.management-utils :refer [MembershipManagementPage]]
+    [broadfcui.common.modal :as modal]
+    [broadfcui.common.style :as style]
+    [broadfcui.common.table.table :refer [Table]]
+    [broadfcui.common.table.style :as table-style]
+    [broadfcui.endpoints :as endpoints]
+    [broadfcui.nav :as nav]
+    [broadfcui.page.groups.create-group :refer [CreateGroupDialog]]
+    [broadfcui.page.workspace.monitor.common :as moncommon]
+    [broadfcui.utils :as utils]
+    ))
+
+
+(react/defc GroupTable
+  {:reload
+   (fn [{:keys [this]}]
+     (react/call :load-data this))
+   :render
+   (fn [{:keys [state this]}]
+     (cond
+       (:error-message @state) (style/create-server-error-message (:error-message @state))
+       (nil? (:groups @state)) [comps/Spinner {:text "Loading groups..."}]
+       :else
+       [Table
+        {:data (:groups @state)
+         :body {:behavior {:reorderable-columns? false}
+                :style table-style/table-light
+                :columns
+                [{:header "Group Name" :initial-width 500 :sort-initial :asc
+                  :sort-by :text
+                  :as-text
+                  (fn [{:keys [group-name]}]
+                    group-name)
+                  :render
+                  (fn [{:keys [group-name access-levels]}]
+                    (if
+                     (contains? (set access-levels) "Owner")
+                      (style/create-link {:text group-name
+                                          :href (nav/get-link :group group-name)})
+                      group-name))}
+                 {:header "Access Levels" :initial-width :auto
+                  :column-data #(clojure.string/join ", " (:access-levels %))}]}
+         :toolbar
+         {:items
+          [flex/spring
+           [comps/Button
+            {:text "Create New Group"
+             :onClick
+             (fn []
+               (modal/push-modal
+                [CreateGroupDialog
+                 {:on-success #(react/call :reload this)}]))}]]}}]))
+   :component-did-mount
+   (fn [{:keys [this]}]
+     (react/call :load-data this))
+   :load-data
+   (fn [{:keys [state]}]
+     (endpoints/get-groups
+      (fn [err-text groups]
+        (if err-text
+          (swap! state assoc :error-message err-text)
+          (let [groups (->>
+                        (map #(identity {:group-name (get-in % [:managedGroupRef :usersGroupName])
+                                         :access-level (:accessLevel %)})
+                             groups)
+                        (group-by :group-name)
+                        (map (fn [[k v]]
+                               {:group-name k :access-levels (sort (map :access-level v))})))]
+            (swap! state assoc :groups groups))))))
+   :-handle-status-change
+   (fn [{:keys [state]} group-name new-status message]
+     (let [group-index (utils/first-matching-index
+                        #(= (:groupName %) group-name)
```

Comment:
Like the other one, I prefer `(comp (partial = group-name) :groupName)`

We could have a utility `(defn is [x] (partial = x))` so that this is `(comp (is group-name) :groupName)`.